R/pkg/NAMESPACE

```diff
@@ -387,8 +387,6 @@ exportMethods("%<=>%",
               "sumDistinct",
               "tan",
               "tanh",
-              "toDegrees",
```

Comment:
@felixcheung to comment on whether we should revisit going ahead and removing these in Spark 3 anyway. We had previously removed them in Pyspark, Scala and kept them in R.

Reply:
they were deprecated  eg. ` .Deprecated("degrees")` in code, I'm ok with removing them

Reply:
`.Deprecated` was just added recently though:
https://github.com/apache/spark/pull/22921/files
This was the previous thread:
https://github.com/apache/spark/pull/22921#discussion_r231125506

Reply:
ah then I"m less sure if we have never release with the deprecation before and go straight to remove
https://github.com/apache/spark/pull/22921#discussion_r230568058

Reply:
Yep I see the argument @felixcheung . I reopened it because we have a similar situation with `SQLContext.createExternalTable`, which was deprecated everywhere but Pyspark. Presumably it was also just an oversight. Looks like we're going to remove that. I'd moderately prefer to remove these for consistency, both in the decision and w.r.t. the `toDegrees` method. Is that persuasive? If you feel fairly strongly about keeping it deprecated, that's OK and I can revert this part.

Reply:
I'm ok with it. I'd prefer we deprecate it first though. Is there going to be 2.4.5 before 3.0?

(I generally would prefer use friendliness - ie. don't break code - than consistency, as not all language API are consistent, esp. with R is always lagging behind...)

Reply:
No prob, I reverted this part of the change.